### DISCUSSION TITLE
Login/user issues

### DISCUSSION THREAD

**Discussion by LinneaEriksson** [vercel/community#4147] [Account Issue]
### Question

Hi, 

I have used vercel for a couple of years. All of a sudden I can't login anymore. My account has been connected to my github for a while and I have a few projects deployed. Now it says that my phone number is "already in use" if i try to login. I then tried with another phone number and now it seems I created a new account cous all the projects I know that I have deployed are gone. Don't know what to do. Help pls :)

**Comment by amyegan** [comment]
Sorry you're having trouble with your account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. If so, please follow those steps.

To open a ticket, please use the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)
